.changeset/eighty-dolls-smile.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-kubernetes-backend': minor
+'@backstage/plugin-kubernetes-common': patch
+---
+
+Check if the namespace exists before listing resources
```

Comment:
It would also be helpful to explain that previously the behavior was to throw an error, but now it returns an empty array 🙂

Reply:
Done https://github.com/backstage/backstage/pull/14714/commits/8aab6a3922dcb6b1afb0c6d91d9c67ea6c00fd12